### ISSUE TITLE
trouble with quotes in writing CSVs and tables

### ISSUE THREAD

**Issue by brimittleman** [swcarpentry/r-novice-inflammation#389]
In reading through the intro R lesson section about reading and writing files, I do not see a discussion about the quote=T/F argument. As a novice, I never understood why the files I wrote had all of the values quoted. Is there a reason this is not discussed or used here?

**Comment by katrinleinweber** [comment]
Hi Briana,

that is an interesting question! Because `git log --oneline | grep "quote"` or `"csv"` don't provide a (to me) obvious answer, I presume it is expected that

a) either [`11-supp-read-write-csv`](https://swcarpentry.github.io/r-novice-inflammation/11-supp-read-write-csv/index.html#the-stringsasfactors-argument) clarifies it, or
b) learners read through `help(read.csv)` and talk about the parameters relevant to them with instructors and helpers.

Do you think that we should point novices to the fact that the `"` or `'` in the .csv being the remnants of R having considered those values as being a string? Equivalent to those quotes being used when defining strings within R.

What other parameter discussion would you find less important? The lesson is already quite full, so we'd have to remove something else if we add something about this.

**Comment by brimittleman** [comment]
I agree that the lesson is quite full. In thinking that way, pointing novices to the fact that quotes are remnants of strings will be sufficient. If they know this then it should be easy to google and fix the problem if they see it.
